protocols/luci-proto-batman-adv/htdocs/luci-static/resources/protocol/batadv_hardif.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+'require uci';
+'require form';
+'require network';
+
+network.registerPatternVirtual(/^bat.+$/);
+
+return network.registerProtocol('batadv_hardif', {
+	getI18n: function() {
+		return _('B.A.T.M.A.N. advanced Interface');
+	},
+	
+	getIfname: function() {
+		return this._ubus('l3_device') || this.sid;
+	},
+	
+	getOpkgPackage: function() {
+		return 'batman-adv';
```

Comment:
If you refer to batman-adv here, then we should also do that in the Makefile
```
- LUCI_DEPENDS:=+kmod-batman-adv +batctl
+ LUCI_DEPENDS:=+batman-adv
```

The packages kmod-batman-adv and batctl are already referenced by batman-adv pacakges

Reply:
OK